Sources/MessagesViewController.swift

```diff
@@ -240,15 +243,23 @@ extension MessagesViewController: UICollectionViewDataSource {
 
 extension MessagesViewController {
 
+
     fileprivate func addKeyboardObservers() {
         NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardDidChangeState), name: .UIKeyboardWillChangeFrame, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(handleTextViewDidBeginEditing), name: .UITextViewTextDidBeginEditing, object: messageInputBar.inputTextView)
     }
 
     fileprivate func removeKeyboardObservers() {
         NotificationCenter.default.removeObserver(self, name: .UIKeyboardWillChangeFrame, object: nil)
```

Comment:
Do we need to remove the notification `.UITextViewTextDidBeginEditing` here?

Reply:
Only for some versions of iOS. Good catch 😉